patches/core-setup/0002-Try-ExcludeFromSourceBuild-in-windowsdesktop.patch

```diff
@@ -0,0 +1,24 @@
+From 1cc0f98a7afd3dd49a95ea8f4a31f80f6805f75c Mon Sep 17 00:00:00 2001
+From: Chris Rummel <crummel@microsoft.com>
+Date: Sun, 4 Aug 2019 21:28:18 -0500
+Subject: [PATCH 2/2] Try ExcludeFromSourceBuild in windowsdesktop
```

Comment:
I like the `[PATCH]` note here, making it clear which commits are from `git am`, but are the numbers useful? Are they kept up to date with some automation?

Reply:
It looks like this was a default that was changed in Git.  We can standardize on one or the other but we'll probably see changes regularly until everyone is on a new enough version.

Reply:
Ah, that's fine then, just wanted to make sure this isn't adding pointless manual work on top of patch maintenance.